cmd/logs/logs.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/config"
+	"github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/k8s/kubeconfig"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+	"github.com/stern/stern/stern"
+)
+
+// LogsOptions options for logs command
+type LogsOptions struct {
+	ManifestPath string
+	Namespace    string
+	Context      string
+	exclude      string
+	Follow       bool
+	Include      string
+	Since        time.Duration
+	Tail         int64
+	Timestamps   bool
+}
+
+func Logs(ctx context.Context) *cobra.Command {
+	options := &LogsOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "logs",
+		Short: "Fetch the logs of a development environment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, cancel := context.WithCancel(ctx)
+			defer cancel()
+
+			manifest, err := contextCMD.LoadManifestWithContext(ctx, contextCMD.ManifestOptions{Filename: options.ManifestPath, Namespace: options.Namespace, K8sContext: options.Context})
+			if err != nil {
+				return err
+			}
+
+			if len(args) > 0 {
+				options.Include = args[0]
```

Comment:
Could this be more than one service name?

Reply:
it's a regexp, you can use regexp for more than one service

Reply:
maybe we can automatically do `okteto logs a b` -> `okteto logs a|b`

Reply:
also, I was considering adding a `--all` flag to list logs from the whole namespace, not just restricted to your git repo